### ISSUE TITLE
RZ4Intune stuck during App Download at 95%

### ISSUE THREAD

**Issue by new-it** [rzander/ruckzuck#269]
Hello,
For a few weeks now, RZ4Intune (1.7.2.5) has been getting stuck very often when creating new apps in Intune. Every time at 95% of the download the complete program hangs. It has to be closed via the task manager, then the same app can be published in Intune. So the download seems to work and the upload process too, but everything together no longer works. This is also very annoying if you want to update or create several apps, then you have to download one after the other and then upload.

I've alreday tried to delete the RZ4Intune.exe, RZ4Intune.exe.config, RZ4Intune.exe.msalcache.bin3 and start from start from scratch but it's not changing anything.

![2024-09-05_08h27_09](https://github.com/user-attachments/assets/125a7416-813a-4613-91d8-19425da22b09)

**Comment by marco-hofmann** [comment]
Same here, sometimes the Queue window just opens and does nothing. Other times it get stuck at random stages. Feels like the App hangs, it can only force close

![image](https://github.com/user-attachments/assets/6837fed8-7c47-453e-89a6-4d9ae417e212)

**Comment by new-it** [comment]
@rzander could you help here please?

**Comment by rzander** [comment]
I cannot reproduce this issue... but to have the same code base, can you try V1.7.3.8...  https://github.com/rzander/ruckzuck/releases/tag/1.7.3.8

**Comment by marco-hofmann** [comment]
> I cannot reproduce this issue... but to have the same code base, can you try V1.7.3.8... https://github.com/rzander/ruckzuck/releases/tag/1.7.3.8

Hi rzander. 
Thank you for your reply. It's the Intune Version that hangs. Now for me it's at 100% but the Powershell script that does the upload to Intune doesn't seem to trigger.

**Comment by new-it** [comment]
Hi rzander,
the new version is not doing anything at all. it only download the application but it's not pushed to intune. and there is also an error shown during the process of app creation.

![image](https://github.com/user-attachments/assets/b2f31b26-d469-489d-804d-350a58db581d)

I've deleted the complete temp folder aldready. In addtion I also created a new folder to store RZ4Intune.exe (1.7.3.8) and RZ4Intune.exe.config (1.7.2.5). I did not find any newer config file on your github site.

**Comment by rzander** [comment]
@new-it , you are right, there was an issue... Please try again with the fixed version: https://github.com/rzander/ruckzuck/releases/download/1.7.3.8/RZ4Intune.exe

**Comment by new-it** [comment]
Still the same :(
![image](https://github.com/user-attachments/assets/e249e7d4-c736-49d8-a5d8-781a1c1dcd68)

**Comment by rzander** [comment]
I cannot reproduce the download Issue:
![image](https://github.com/user-attachments/assets/3cccab26-e135-4a74-aac4-89912b7e0ab0)

RZ4Intune.exe running in a Windows11 24H2 Sandbox with direct Internet connection...
